_docs/management/space-egress.md

```diff
@@ -17,7 +17,7 @@ A summary of each of the ASGs that can be applied to your space are as follows:
 | :-------- |  :-: | :--:  | :--: | :-------: | :---------------------: | :-----------------: | :---------------: |
 | `closed-egress`     | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ✅ |
 | `restricted-egress` | ❌ | ❌ | ✅  | ✅ | ✅ | ✅ | ✅ |
-| `public-egress`     | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | 
+| `public-egress`     | ✅ | ✅ | ✅ | ❌ | ❌ | ❌ | ✅ | 
```

Comment:
Also, I think the CF security group that actually gets applied is `public_networks_egress`. So I'm just wondering `public-egress` means for customers

Reply:
Yeah these arent the names of the security groups but rather the term for the type of access.